lumberdev/atelier
# Bugfix/campaign form TS error

### WHAT IT DOES:

- Fixed campaign form type errors
- Fixed prettier plugin missing

### WHAT ISN'T INCLUDED:

- The bug affecting the save-bar is still present and being working on in a separate PR. This bug is where the save-bar doesn't appear on changing inputs not controlled by `react-hook-form`, which includes:
  - Status 
  - Image
  - Cart Image Style
  - Selected Products
  - Selected Collections

### WHAT TO TEST:

- Ensure DB is reset & synced 
- Create a campaign
- Test if you can change & save the fields not listed above
- Test that prettier works correctly in the IDE
